templates/include/glean/dot_separated_short_id.1.schema.json

```
@@ -1,5 +1,5 @@
 {
   "type": "string",
-  "pattern": "^[a-z_][a-z0-9_]{0,29}(\\.[a-z_][a-z0-9_]{0,29})+$",
-  "maxLength": 61
+  "pattern": "^[A-Za-z_][A-Za-z0-9_\\.]*$",
```

this probably should not have the uppercase letters part (and `+` instead of `*` to have at least one character)

```suggestion
  "pattern": "^[a-z_][a-z0-9_\\.]+$",
```